components/coding/studentPortal/ProfileDetailCard.tsx

```diff
@@ -10,57 +13,61 @@ type ProfileDetailCard = {
 function ProfileDetailCard({
   avatar,
   name,
+  link,
   joinDate,
   badges,
   currentFocus,
   nextGoal,
 }: ProfileDetailCard) {
+  let profile = "profile/" + link;
```

Comment:
this can actually be const because the variable is never reassigned in this component, but it works either way